week_4/quiz-trivia-app/src/app/storage.service.ts

```diff
@@ -0,0 +1,28 @@
+export class StorageService {
+  // Save data to localStorage
```

Comment:
remove all comments, the logic isnt that complex to warrant comment for explaination.